Conclude 3TWC (#4986)
ppy/osu-wiki: wiki/Tournaments/3TWC/1/en.md

![3TWC logo](img/logo.png)

> The **3 Digit Taiko World Cup** (***3TWC***) was a worldwide country-based double-elimination 2v2 osu!taiko tournament hosted by ![][flag_AR] [Vaf](https://osu.ppy.sh/users/12589048). Only players ranked between #100 and #999 can participate. It was the first installment of the 3 Digit Taiko World Cup.

Reviewer: ```suggestion
The **3 Digit Taiko World Cup** (***3TWC***) was a worldwide country-based double-elimination 2v2 osu!taiko tournament hosted by ![][flag_AR] [Vaf](https://osu.ppy.sh/users/12589048). Only players ranked between #100 and #999 could participate. It was the first installment of the 3 Digit Taiko World Cup.
```

- Livestream
  - [Main channel](https://www.twitch.tv/3twc)
  - [Secondary channel](https://www.twitch.tv/3twc_b)

>     - *Note: The second channel's purpose is to stream other match if there are more than one match happening at the same time.*

Reviewer: ```suggestion
    - *Note: The second channel's purpose was to stream the other match if there were more than one matches happening at the same time.*
```

| ![][flag_US] | **United States** | **[cheese salad](https://osu.ppy.sh/users/6349821)**, [AuroraPhasmata](https://osu.ppy.sh/users/13664116), [ChurroChef](https://osu.ppy.sh/users/9258564), [Ethaaaan](https://osu.ppy.sh/users/9536977), [R J](https://osu.ppy.sh/users/6490509) |
| ![][flag_CR] ![][flag_VE] | **Venezuela Rica** | **[Colorojo12](https://osu.ppy.sh/users/5136821)**, [\[\_Chichinya\_\]](https://osu.ppy.sh/users/2140739), [Hotman](https://osu.ppy.sh/users/7902082), [Nutellitah](https://osu.ppy.sh/users/10613075), [puijela](https://osu.ppy.sh/users/12687433) |

> 

Reviewer: escape the code tick in L206
